cpp/server/ct-mirror.cc

```diff
@@ -134,6 +135,7 @@ using std::shared_ptr;
 using std::string;
 using std::thread;
 using std::unique_ptr;
+using util::HexString;
```

Comment:
Not used? And I'm guessing that the `#include "util/util.h"` is for this, so unneeded?